### PR TITLE
[NVIDIA Container] Fix `PATH` to include NVIDIA binaries

### PR DESCRIPTION
**Pull request by DEKHTIARJonathan**
This branch addresses and fix a `PATH` issue in the NVIDIA containers.

```sh
docker run -it \
    --gpus=all \
    gcr.io/iree-oss/nvidia-bleeding-edge bash

# %%%% Container is starting  %%%% 

# As you can see `nvcc` is not in the PATH
root@12c0c715d753:/ nvcc --version
bash: nvcc: command not found

# Now let's fix the issue "by hand" and try again
root@12c0c715d753:/ PATH=${PATH}:/usr/local/cuda/bin/
root@12c0c715d753:/ nvcc --version

nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Wed_Sep_21_10:33:58_PDT_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0
```

This simple PR fixes the problem.

@ScottTodd & @GMNGeoffrey  for review

### CODE REVIEW

File: build_tools/docker/dockerfiles/nvidia.Dockerfile [hunk 1]
```diff
@@ -57,3 +57,6 @@ RUN wget https://developer.download.nvidia.com/compute/cuda/11.6.0/local_install
   && apt-key add /var/cuda-repo-debian11-11-6-local/7fa2af80.pub \
   && apt-get update \
   && apt-get -y install cuda-toolkit-11-6
+
+# Adding CUDA binaries to Path
+ENV PATH=${PATH}:/usr/local/cuda/bin/
```
Comment (GMNGeoffrey) [inline build_tools/docker/dockerfiles/nvidia.Dockerfile]: Nit: please add trailing newline

```suggestion
ENV PATH=${PATH}:/usr/local/cuda/bin/

```

**Review by GMNGeoffrey — COMMENTED** [review]
Oh if it's just an issue of it failing when you directly invoke tools from the command line, it's not surprising that this wasn't a problem in CI (which is what these containers were created for). Adding to PATH seems fine though.

I'll rebuild the containers and push here

File: build_tools/docker/dockerfiles/nvidia.Dockerfile (hunk 1 above)
Comment (ScottTodd) [reply build_tools/docker/dockerfiles/nvidia.Dockerfile]: These VSCode settings are helpful for formatting, FYI:
```json
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
    "files.trimTrailingWhitespace": true,
```

Comment (DEKHTIARJonathan) [reply build_tools/docker/dockerfiles/nvidia.Dockerfile]: Added to my dev space. Thanks @ScottTodd

**Review by ScottTodd — APPROVED** [review]

**Comment by DEKHTIARJonathan** [comment]
@ScottTodd how does it work to merge to PR once approved ?

**Comment by ScottTodd** [comment]
> @ScottTodd how does it work to merge to PR once approved ?

https://github.com/openxla/iree/blob/main/CONTRIBUTING.md#merging

> It is assumed that the PR author will merge their change unless they ask someone else to merge it for them (e.g. because they don't have write access).

(I thought Geoffrey might click merge, but I can do it)

**Pull request merged**